src/test/java/pages/TutorialspointPage.java

```diff
@@ -0,0 +1,46 @@
+package pages;
+
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class TutorialspointPage extends AbstractPage {
+
+    private static final String TUTORIALSPOINT_URL = "https://www.tutorialspoint.com/";
+
+    @FindBy(id = "gsc-i-id1")
+    private WebElement searchInputArea;
+
+    @FindBy(xpath = ".//div[contains(@id, 'resInfo')]")
+    private WebElement searchResults;
+
+    public TutorialspointPage(WebDriver driver) {
+        super(driver);
+    }
+
+    public TutorialspointPage openPage() {
+        driver.get(TUTORIALSPOINT_URL);
+        driver.manage().window().maximize();
+        waitForPageLoad(driver);
+
+        return this;
+    }
+
+    public TutorialspointPage searchForPhrase() {
+        String searchValue = "submit() vs click()";
+        waitForPageLoad(driver);
+        searchInputArea.sendKeys(searchValue, Keys.ENTER);
+
+        return new TutorialspointPage(driver);
+    }
+
+    public int countTutorialspointSearchResults() {
+        waitForPageLoad(driver);
+        elementIsPresent(searchResults);
+        String searchResultsText = searchResults.getText();
```

Comment:
more elegant solution is:
except lines 41-43, use - 
return searchResults.getText().split(" ")[1]

Reply:
Thank you, absolutely agree.
I added replacement I used before to avoid failing of test because of comma after thousands (4,150 =>4150).

return Integer.parseInt(searchResults.getText().split(" ")[1].replaceAll("[\\D]", ""));